iPetCare/Application/Dtos/Species/SpeciesCreateDtoRequest.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.ComponentModel.DataAnnotations;
+
+namespace Application.Dtos.Species
+{
+    public class SpeciesCreateDtoRequest
+    {
+        [Required]
+        public string Name { get; set; }
```

Comment:
Jest git, ewentualnie można nałożyć ograniczenie odnośnie długości. Jak nie ograniczymy tego na froncie lub w kontrolerze to odbije się od bazy danych z errorem. Niby ok ale możemy tego łatwo uniknać stosujać własnie ten Data Annotation. 
Data annotation to np. właśnie użyty tutaj [Required] :) 

https://www.entityframeworktutorial.net/code-first/maxlength-minlength-dataannotations-attribute-in-code-first.aspx